src/components/Sum.jsx

```diff
@@ -1 +1,13 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({
+  a = 0,
+  b = 0,
+  text = `Sum of ${a} and ${b} is ${a + b}`,
```

Comment:
you didn't accept this string in props - so move it inside component as a variable or use in return directly

Reply:
fixed